common/lib/xmodule/xmodule/js/src/video/04_video_control.js

```diff
@@ -296,7 +296,10 @@ function () {
     }
 
     function updateVcrVidTime(params) {
-        this.videoControl.vidTimeEl.html(Time.format(params.time) + ' / ' + Time.format(params.duration));
+        var duration = (this.config.endTime !== null) ? this.config.endTime : params.duration;
```

Comment:
Yep, 0 is a legitimate value for endTime. Depending on the value of `startTime`, it may get changed `null` by the front-end before further processing is done.